### ISSUE TITLE
Package installation fails in Windows

### ISSUE THREAD

**Issue by gitname** [patientslikeme/react-calendar-heatmap#58]
### Background Info

The `package.json` file in this package contains the following lines, among others:

```json
  "scripts": {
    "build": "NODE_ENV=production webpack",
```

My computer is running Windows 7 Professional 64-bit.

### The Problem

When I run `$ npm install` ([which runs `$ npm build`](https://docs.npmjs.com/cli/build#description), which maps to the command shown above), the Windows Command Prompt tries to run a command named `NODE_ENV`. The Windows Command Prompt does not find such a command; so, it shows the following error message:

```bash
'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-calendar-heatmap@1.6.1 build: `NODE_ENV=production webpack`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-calendar-heatmap@1.6.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\SomeUser\AppData\Roaming\npm-cache\_logs\2017-11-25T07_30_29_787Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

### A Solution... for Windows (at least)

The above error can be avoided by changing the `build` command to the one shown below, which uses `set` and `&&`.

```json
  "scripts": {
    "build": "set NODE_ENV=production && webpack",
```

With that change in place, I can build the package at the Windows Command Prompt with no errors.

Although that fixes the build process in Windows, it may break it on Mac/Linux (I don't know whether it does or not).

### A Possible Solution... for Windows, Mac, and Linux

There is an NPM package called [cross-env](https://www.npmjs.com/package/cross-env) that I think could be used to solve this problem.

The package's README says:

> cross-env makes it so you can have a single command without worrying about setting or using the environment variable properly for the platform. Just set it like you would if it's running on a POSIX system, and cross-env will take care of setting it properly.

**Comment by gitname** [comment]
This issue is fixed in commit 648f315d62dcfb5275db110c755c94a89feabe48 (part of *Pull Request #61*).

**Comment by okize** [comment]
Fixed by https://github.com/patientslikeme/react-calendar-heatmap/pull/61
